assets/manifest.json

```diff
@@ -6,6 +6,12 @@
 			],
 			"main": true
 		},
+		"customizer.js": {
+			"files": [
+				"scripts/customizer.js"
+			],
+			"main": true
```

Comment:
Can you please remove this line? It's only needed on the main.js file. Otherwise, Bower assets that aren't overwritten will be added to this one as well.